setup.cfg

```diff
@@ -69,6 +69,7 @@ filterwarnings =
     # Suppress deprecation warnings in other packages
     ignore:lib2to3 package is deprecated::scspell
     ignore:SelectableGroups dict interface is deprecated::flake8
+    ignore:The 'asyncio_mode' default value will change to 'strict' in future:DeprecationWarning:pytest
```

Comment:
I'm not totally familiar with the syntax here, but this looks different from the ones around it.  In particular, that `DeprecationWarning` may be a typo?  Given the other lines, I would have expected:

```
    ignore:The 'asyncio_mode' default value will change to 'strict' in future::pytest
```

But I could be totally offbase.

Reply:
This is warnings filter syntax: https://docs.python.org/3/library/warnings.html#the-warnings-filter

The one I'm adding is just a little more specific. I could drop the `DeprecationWarning` and it would still work fine.